src/Core/src/Platform/iOS/TextViewExtensions.cs

```diff
@@ -58,6 +58,8 @@ public static void UpdateFont(this UITextView textView, ITextStyle textStyle, IF
 			var font = textStyle.Font;
 			var uiFont = fontManager.GetFont(font, UIFont.LabelFontSize);
 			textView.Font = uiFont;
```

Comment:
Can we override the Font property and set the placeholder label to match? Not sure if that works...

Reply:
took your advice and used the override of Font on MauiTextView.

Didn't pass all the properties because I don't think that happens in all platforms, that would need some discussion I think, maybe opening a new issue related with that.

Reply:
looks like a I needed a refresh...